ee/vellum_ee/workflows/display/nodes/vellum/base_adornment_node.py

```diff
@@ -83,18 +88,25 @@ def exec_body(ns: Dict):
                     # we skip it, since it should not be included in the adorned node's output displays
                     continue
 
-                if old_output not in inner_cls.output_display:
-                    # If the adorned node doesn't have an output display for this output, we skip it
-                    continue
-
-                inner_cls.output_display[new_output] = inner_cls.output_display.pop(old_output)
+                if old_output not in wrapped_node_display_class.output_display:
+                    # If the adorned node doesn't have an output display defined for this output, we define one
+                    wrapped_node_display_class.output_display[new_output] = NodeOutputDisplay(
+                        id=wrapped_node_class.__output_ids__[old_output.name],
+                        name=old_output.name,
+                    )
+                else:
+                    wrapped_node_display_class.output_display[new_output] = (
+                        wrapped_node_display_class.output_display.pop(old_output)
+                    )
```

Comment:
this is the actual change that enables. The rest is just renames and strict typing for clarity